packages/devtools_app_shared/lib/src/ui/theme/ide_theme.dart

```diff
@@ -24,18 +24,20 @@ final class IdeTheme {
     bool? isDarkMode,
   }) : _isDarkMode = isDarkMode;
 
-  final Color? backgroundColor;
-  final Color? foregroundColor;
-  final double fontSize;
+  Color? backgroundColor;
+  Color? foregroundColor;
+  double fontSize;
   final EmbedMode embedMode;
-  final bool? _isDarkMode;
+  bool? _isDarkMode;
```

Comment:
why do these need to be non-final?

Reply:
I think this was a change I made while testing that should be reversed.